etc/crds/endpoints.crd.yaml

```diff
@@ -37,6 +37,14 @@ spec:
     shortNames:
       - ep
       - eps
+  validation:
+    openAPIV3Schema:
+      properties:
+        backends:
+          type: array
+          items:
+            type: string
+          minItems: 0
```

Comment:
I'll remove this.
This was originally needed since when system services come up, the bouncers are not yet provisioned.
Thus we had to rely on the bouncers to call the update RPCs. 
The kopf retry mechanism fixes this.